tests/test_convert.py

```diff
@@ -1081,3 +1082,57 @@ def test_convert_hte_cellprofiler_csv(
 
     # clean up the parsl config for other tests
     parsl.clear()
+
+
+def test_cellhealth_preset(
```

Comment:
I'm not sure if you do this elsewhere (I'm sure you do, actually), but it might be nice to include an explicit join test for this preset as well.

Specifically, can you load each sqlite table individually, merge, and then compare to the convert approach? (you'll probably have to replace some `nan` values) but I think it would be good to confirm accurate join.

Reply:
Great point, thanks! I'll add this in.